code/game/objects/items/devices/flashlight.dm

```diff
@@ -205,6 +205,17 @@
 	brightness_on = 5 // A little better than the standard flashlight.
 	hitsound = 'sound/weapons/genhit1.ogg'
 
+/obj/item/flashlight/seclite/flashdark

```

Comment:
flashdarks already exist under the `/obj/item/flashlight/flashdark` subtype

Reply:
reverted adding the flashdark